src/Microsoft.DotNet.Build.Tasks/UpdateVSConfigurations.cs

```diff
@@ -40,14 +40,14 @@ public override bool Execute()
                 Log.LogMessage($"Updating {projectFile}");
 
                 var project = ProjectRootElement.Open(projectFile);
-                ICollection<ProjectPropertyGroupElement> propertyGroups;
-                var actualConfigurations = GetConfigurationFromPropertyGroups(project, out propertyGroups);
+                ICollection<ProjectPropertyGroupElement> propertyGroups = GetPropertyGroupsToRemove(project);
+                var actualConfigurations = GetConfigurationsFromProperty(project);
 
                 bool addedGuid = EnsureProjectGuid(project);
```

Comment:
Does this mean we are still going to keep project guid's in the project? Is that still our plan to keep them stable?

Reply:
I debated on removing them now or not.  My thinking is to leave them for now, in the hopes of getting the first wave of SDK-style changes merged - have to draw the line somewhere.

My plan for removing them, and keeping the GUIDs stable, is to read the GUIDs from the `.sln` file, and associate that GUID with the project.  However, parsing the full `.sln` file was something I didn't want to try to tackle right now. Thus why I left them in.

Reply:
Sounds good. We could also hash the project name to get a stable guid.